docker-compose.test.yml

```diff
@@ -1,3 +1,3 @@
 sut:
-  build:
-  command: ./run_tests.sh
\ No newline at end of file
+  build: .
+  command: ./run_test.sh
```

Comment:
Still a mismatch here in the file name.  One has an underscore and the file name has a dash.  Make this change in your Aashirya branch and `git push origin Aashirya`

from:

```
  command: ./run_test.sh
```

to 

```
  command: ./run-test.sh
```